### DISCUSSION TITLE
Lock card - hide right button

### DISCUSSION THREAD

**Discussion by mvereecken** [piitaya/lovelace-mushroom#595] [Q&A]
Is there a way to hide the right lock icon (the actual action button) in the lock card? I think it's not really needed if you use the card as an action to lock/unlock. Because of the right lock icon:
1) it takes a lot more space, so I cant have 2 locks next to eachother, without the label becoming unreadable
![image](https://user-images.githubusercontent.com/3824067/177924005-e40fddfc-5264-40c0-96f8-711e70b54eff.png)
2) Smaller issue with this: it could be a bit confusing, 2 lock icons, 1 is for the state, the other for the action.
Thanks for all the great work with the mushroom project :-)

**Comment by alessandroias** [comment]
You can use a template card
